src/builder/Actions.sol

```diff
@@ -461,6 +471,74 @@ library Actions {
         return (quarkOperation, action);
     }
 
+    function cometWithdrawAsset(CometWithdraw memory cometWithdraw, PaymentInfo.Payment memory payment)
+        internal
+        pure
+        returns (IQuarkWallet.QuarkOperation memory, Action memory)
+    {
+        bytes[] memory scriptSources = new bytes[](1);
+        scriptSources[0] = type(CometWithdrawActions).creationCode;
+
+        Accounts.ChainAccounts memory accounts =
+            Accounts.findChainAccounts(cometWithdraw.chainId, cometWithdraw.chainAccountsList);
+
+        Accounts.AssetPositions memory assetPositions =
+            Accounts.findAssetPositions(cometWithdraw.assetSymbol, accounts.assetPositionsList);
+
+        Accounts.QuarkState memory accountState =
+            Accounts.findQuarkState(cometWithdraw.withdrawer, accounts.quarkStates);
+
+        bytes memory scriptCalldata;
+        if (Strings.stringEqIgnoreCase(cometWithdraw.assetSymbol, "ETH")) {
+            // XXX handle unwrapping ETH
```

Comment:
cc: @cwang25

Reply:
I'm thinking if it is necessary to unwrap ETH here? 
Since we treat ETH/WETH as the same asset in Legend app (thus need auto wrapper/unwrapper feature). 
Part of me feel like we can leave it as WETH after withdraw.  If user want to transfer ETH out or something else later on ... the auto unwrapper should handle unwrapping WETH freely by then